### PR TITLE
tweego: init at 2.1.1

### PR DESCRIPTION
**Pull request by ChrisPWill**
## Description of changes

Tweego is a compiler for Twine/Twee projects - a popular tool used for authoring interactive text fiction.
https://www.motoslave.net/tweego/

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
Compiled a basic project following instructions on website and it all worked as expected.
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by redyf** [comment]
Result of `nixpkgs-review pr 326795` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tweego</li>
  </ul>
</details>

**Review by redyf — APPROVED** [review]

**Comment by ChrisPWill** [comment]
@NixOS/nixpkgs-merge-bot merge

File: pkgs/by-name/tw/tweego/package.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  proxyVendor = true;
+  vendorHash = "sha256-1O27CiCXgrD0RC+3jrVxAiq/RnI2s1FW2/uoBAd1fF8=";
+
+  preBuild = ''
+    go mod tidy
+  '';
+
+  meta = {
+    description = "Tweego is a free (gratis and libre) command line compiler for Twine/Twee story formats, written in Go.";
```
Comment (pbsds) [inline pkgs/by-name/tw/tweego/package.nix:25]:
```suggestion
    description = "Free (gratis and libre) command line compiler for Twine/Twee story formats, written in Go";
```

**Review by pbsds — COMMENTED** [review]
Please also reorder the two commits, so the maintainer addition comes first

File: pkgs/by-name/tw/tweego/package.nix (hunk 1 above)
Comment (ChrisPWill) [reply pkgs/by-name/tw/tweego/package.nix]: Thanks! I've re-ordered the commits (updating code as necessary) and implemented your suggestion. Makes a lot of sense to keep to the point.

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**